jupyter/nmm-game-utils.ipynb

```diff
@@ -49,16 +37,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Calculates the opponent of the given player\n",
-    "def opponent(p):\n",
-    "    return 'b' if p == 'w' else 'w'"
+    "START_STATE = ((9, 9), (\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '),\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '),\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ')\n",
+    "))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Stones"
+    "## Funktionen um mit Steinen zu interagieren"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### hasPlaceableStones\n",
+    "\n",
+    "Die Funktion `hasPlaceableStones` überprüft, ob ein Spieler für einen Zustand noch zusetzende Steine auf dem Stapel (_stash_) besitzt. Die Funktion hat zwei Argumente:\n",
+    "\n",
+    "- `s` ist ein Zustand eines Spiels;\n",
+    "- `p` ist ein Spieler.\n",
+    "\n",
+    "Die Funktion gibt ein booleschen Wert zurück."
```

Comment:
Da ich kein Kommentar auf nicht geändertem Code machen kann, kommt der hier in:

Meint Stroetmann nicht, dass wir die Dokumentation innerhalb des Codes auf ein Minimum reduzieren sollten? Hier ist diese beispielsweise nicht nötig.

Reply:
Ja ich glaub ich hab es generell vergessen zu entfernen 🤦‍♂️